test/ui-test/src/screens/StatusCommunityScreen.py

```diff
@@ -228,36 +238,42 @@ def verify_category_contains_channels(self, community_category_name, community_c
                 verify_failure("Channel " + channel_name + " should be checked in category " + community_category_name)
         comma = ", "
         verify(len(split) == 0, "Channel(s) " + comma.join(split) + " should not be checked in category " + community_category_name)
-
-    def edit_community(self, new_community_name: str, new_community_description: str, new_community_color: str):
+    
+    def open_edit_community_by_community_header(self):
         click_obj_by_name(CommunityScreenComponents.COMMUNITY_HEADER_BUTTON.value)
+        time.sleep(0.5) # wait for UI to load
```

Comment:
If you can, try to maybe `wait for object and get` to not have to rely on a sleep.

Reply:
This doesn't work. `click_obj_by_name` inside calls  `waitForObject` which has default timeout == 20 sec.
I think the problem is in `getattr`, because you are trying to getattr for non created object and then you are waiting for this object.
So we need to make some wrapper for `getattr` (for example with tries and sleep for 0.1 sec), but this will impact all tests and I would like not to do it in scope of this task